src/transformers/models/mistral/modeling_mistral.py

```diff
@@ -364,7 +364,7 @@ def forward(
         if past_key_value is not None:
             # Activate slicing cache only if the config has a value `sliding_windows` attribute
             if hasattr(self.config, "sliding_window") and kv_seq_len > self.config.sliding_window:
-                slicing_tokens = kv_seq_len - self.config.sliding_window
+                slicing_tokens = 1 - self.config.sliding_window
```

Comment:
Can you explain this? Wouldnt this make slicing_tokens negative?

Reply:
It's the start index for slicing tokens. Line 375 check whether the size is self.config.sliding_window - 1. Slicing from 1-self.config.sliding_window  will ensure this.